README.md

```diff
@@ -3,359 +3,39 @@
 
-If you are developing a font project publicly with Github (or a similar host) you can set up a Continuous Integration service (like [Travis](https://www.travis-ci.org)) to run Font Bakery on each commit, so that with each update all checks will be run on your files.
-
-This project is currently maintained by Felipe Corrêa da Silva Sanches <juca@members.fsf.org>, with very frequent contributions from Lasse Fister, Marc Foley and Dave Crossland.
-
-Font Bakery is not an official Google project, and Google provides no support for it.
-
-## Command Line Tools
-
-Since v0.3.3 Font Bakery is solely focused on automated quality checking of font families.
-
-Font Bakery up to v0.3.2 provided some auxiliary scripts, and these have been moved into a separate "Google Fonts Tools" python package `gftools` at <https://github.com/googlefonts/tools> and packaged at <https://pypi.python.org/pypi/gftools>
-
-Installing the latest version of the auxiliary scripts should be as easy as:
-
-    pip install gftools --upgrade
-
-## Web Usage
-
-A web dashboard for monitoring check-results of the full Google Fonts collection (or possibly other collections of font families) was initialy developed in this repository, but is now available at <https://github.com/googlefonts/fontbakery-dashboard>
-
-(There is an older drag-and-drop web app still hosted at <https://fontbakery.appspot.com> but this is not actively maintained.)
-
-## Command Line Usage
-
-Install Font Bakery as a package, following the [installation instructions](https://github.com/googlefonts/fontbakery#install), and you will have a `fontbakery` command in your `$PATH`
-
-This has several subcommands, described in the help function:
-
-```
-$ fontbakery -h
-usage: fontbakery [-h] [--list-subcommands] subcommand
-
-Run fontbakery subcommands:
-    build-contributors
-    check-collection
-    check-googlefonts
-    check-noto-version
-    check-specification
-    generate-glyphdata
-
-Subcommands have their own help messages. These are usually accessible with the -h/--help flag positioned after the subcommand.
-I.e.: fontbakery subcommand -h
-
-positional arguments:
-  subcommand          the subcommand to execute
-
-optional arguments:
-  -h, --help          show this help message and exit
-  --list-subcommands  print the list of subcommnds to stdout, separated by a space character. This is usually only used to generate the shell completion code.
-```
-
-### fontbakery check-googlefonts
-
-This is the central script to run the FontBakery suite of checks.
-It runs the checks (`specifications/googlefonts.py`) that we use for QA of <https://github.com/google/fonts>
-
-To run the checks on some fonts:
-
-    $ cd ~/path/to/fontproject/
-    $ fontbakery check-googlefonts *.ttf
+The project was initiated by Dave Crossland in 2013 to accelerate the onboarding process for Google Fonts. 
+In 2017 Lasse Fister rewrote it into a modern, modular architecture suitable for both individuals and large distributors. 
```

Comment:
I think my name should be mentioned here as well, Dave. ;-)

Reply:
"and large distributors. Felipe Sanches has maintained the check contents since 2016."?

Reply:
yep! sounds good